src/Lynx/List.purs

```diff
@@ -75,8 +80,39 @@ type RenderView
 type RenderColumn
   = { name :: Lynx.Expr.ExprType
     , value :: Lynx.Expr.Key
+    , width :: Maybe Size
     }
 
+-- | Representation of various sizing grades for use in UI
+data Size
+  = Small
+  | Medium
+  | Large
+
+derive instance eqSize :: Eq Size
+
+derive instance genericSize :: Generic Size _
+
+instance showSize :: Show Size where
+  show = genericShow
+
+instance encodeJsonSize :: Data.Argonaut.EncodeJson Size where
```

Comment:
Can we drop the `DecodeJson`/`EncodeJson` instances in favor of using codecs? The intent behind using the codecs is that we then don't need to write a roundtrip test for the JSON side of things. We should have enough confidence that what we're encoding and decoding works with each other.

If we go with the `DecodeJson`/`EncodeJson` approach, we have to validate that those line up with a roundtrip test like we've been doing for all the JSON stuff in `Lynx.Expr` and `Lynx.Form`. Which  means that we'll have to add a test suite for `Lynx.List` round tripping.